README.md

```diff
@@ -20,6 +20,14 @@ GitHub](https://github.com/balena-io/balena-cli/), and your contribution is also
 Check the [balena CLI installation instructions on
 GitHub](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md).
 
+## Development
+
+You can build the CLI for testing during development like so:
+
+`npm install && npm run build`
+
+After the build is completed, run `bin/balena` to test your changes.
```

Comment:
Towards the end of the README file, there is already a link to the [CONTRIBUTING.md](https://github.com/balena-io/balena-cli/blob/master/CONTRIBUTING.md) document that contains hints and instructions for the developing and submitting pull requests:

> https://github.com/balena-io/balena-cli/blob/master/README.md#contributing-including-editing-documentation-files
> **Contributing (including editing documentation files)**
> Please have a look at the [CONTRIBUTING.md](https://github.com/balena-io/balena-cli/blob/master/CONTRIBUTING.md) file for some guidance before submitting a pull request or updating documentation (because some files are automatically generated). Thank you for your help and interest!

Reply:
How about, instead of adding this `Development` section, edit the `Contributing` section to include the word 'development' somewhere, such that someone searching the doc for 'dev' with CTRL-F would find that section?